newsfragments/41453.significant.rst

```diff
@@ -0,0 +1,11 @@
+Removed legacy scheduling arguments on DAG
+
+The ``schedule_interval`` and ``timetable`` arguments are removed from ``DAG``.
+
+The ``schedule_interval`` _attribute_ has also been removed. In the API, a new
```

Comment:
Why `timetable_summary` and not `schedule`?

Reply:
This matches the column name on `DagModel`. We can’t just call this `schedule` since there’s also a field `timetable_summary`. I guess if we really want to we can call this `schedule_summary` (and also rename the description field), but I prefer *timetable* here since it does not represent datasets.

Reply:
As a daily user, most of jobs that i run did not require 'dataset' features at all and when i got to know that feature it was because of my interest and at that time years was passed about when i start using Apache Airflow. I hear similar convos around, so i discourage to just flying away / replacing 'schedule_interval' that almost all DAGs are using in order to provide some meaning integrity that about a feature not yet adopted for now. @kaxil

Reply:
I don’t understand what the comment is trying to say, to be honest.